Sources/Tentacle/Client.swift

```diff
@@ -337,12 +343,13 @@ public final class Client {
     }
 
     /// Fetch the content for a path in the repository
-    public func content(atPath path: String, in repository: Repository) -> SignalProducer<(Response, Content), Error> {
-        return fetchOne(.content(owner: repository.owner, repository: repository.name, path: path))
+    public func content(atPath path: String, in repository: Repository, at ref: String? = nil) -> SignalProducer<(Response, Content), Error> {
+        return fetchOne(.content(owner: repository.owner, repository: repository.name, path: path, ref: ref))
     }
 
-    public func create(file: File, atPath path: String, in repository: Repository) -> SignalProducer<(Response, FileResponse), Error> {
-        return send(file, to: .content(owner: repository.owner, repository: repository.name, path: path), using: .put)
+    /// Create a file in a repository
+    public func create(file: File, atPath path: String, in repository: Repository, in branch: String? = nil) -> SignalProducer<(Response, FileResponse), Error> {
```

Comment:
These new parameters above should follow the API Design Guidelines: https://swift.org/documentation/api-design-guidelines/

> Compensate for weak type information to clarify a parameter’s role.